src/c/README.md

```diff
@@ -0,0 +1,118 @@
+## C Language Basic Syntax Review
+C language is one of the most popular general-purpose programming language developed by Dennis Ritchie at Bell laboratories for UNIX operating system. The initial release of C Language was in the year 1972. Most of the desktop operating systems are written in C Language.

```

Comment:
Can we update 'most popular general-purpose programming language' to 'most popular general-purpose programming languages'?

Also, can we add a 'the' before 'UNIX operating system'?